CHANGELOG/CHANGELOG-1.2.md

```diff
@@ -1,5 +1,23 @@
-# v1.2.1 - Changelog since v1.2.0
+# v1.2.2 - Changelog since v1.2.1
+
+## Changes by Kind
+
+### Feature
+
+- Update base image to buster-1.5.0. ([#755](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/755), [@mattcary](https://github.com/mattcary))
```

Comment:
we now are using buster-1.6.0 -> https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/release-1.2/Dockerfile#L24
this PR(https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/760) update that but did not have a release notes :(

Reply:
updated the PR to add the release notes

Reply:
udpated